From 83b45279d16ce25d3134c2549c76dcf3624f6868 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 14 Mar 2025 21:20:49 -0700
Subject: [PATCH 10/21] fix patchelf not found

Set use_default_shell_env=True [1] use the default shell environment
PATH to provide command patchelf

[1] https://bazel.build/rules/lib/builtins/actions#run

Upstream-Status: Inappropriate [wr-installer specific]

Rebase  to 2.19
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorflow/tools/pip_package/utils/tf_wheel.bzl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tensorflow/tools/pip_package/utils/tf_wheel.bzl b/tensorflow/tools/pip_package/utils/tf_wheel.bzl
index 057779e946e..01effece9d2 100644
--- a/tensorflow/tools/pip_package/utils/tf_wheel.bzl
+++ b/tensorflow/tools/pip_package/utils/tf_wheel.bzl
@@ -119,6 +119,7 @@ def _tf_wheel_impl(ctx):
         inputs = srcs + headers + xla_aot,
         outputs = [output_file],
         executable = executable,
+        use_default_shell_env = True,
     )
     return [DefaultInfo(files = depset(direct = [output_file]))]
 
-- 
2.25.1

